Makefile

```diff
@@ -120,6 +120,13 @@ coverage: ## show the coverage report
 	# coverage report
 	echo no-op coverage report
 
+
 .PHONY: clean
 clean: ## clean up the environment by deleting the .venv, dist, eggs, mypy caches, coverage info, etc
-	rm -rf .venv $(DEPS) dist *.egg-info .mypy_cache build .pytest_cache .coverage runinfo $(WORKQUEUE_INSTALL)
+	rm -rf .venv $(DEPS) dist *.egg-info .mypy_cache build .pytest_cache .coverage runinfo_* $(WORKQUEUE_INSTALL)
```

Comment:
This line is merged badly - a recent PR #3236 removed that _ on the end of runinfo